tests/compositional/test_materials.py

```diff
@@ -17,7 +17,9 @@
 import pytest
 
 import porepy as pp
+
 from porepy.examples.flow_benchmark_2d_case_1 import FractureSolidConstants
+from porepy.compositional.materials import FractureDamageSolidConstants
```

Comment:
This is an awkward placement for the materials objects. I guess we all agree on that one.

Reply:
Indeed. Nothing to do here, though, right?

Reply:
Yes. User project? Resolve anyhow.